.github/workflows/checks.yml

```diff
@@ -0,0 +1,13 @@
+name: checks
+run-name: Spot check for errors
+on:
+  - push
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: curl -L https://nixos.org/nix/install | sh
+      - run: |
+          . /home/runner/.nix-profile/etc/profile.d/nix.sh
+          scripts/ci_check.sh
```

Comment:
can you make these 2 separate runs?

Reply:
nevermind, `uses: cachix/install-nix-action@v20` fixes this

Reply:
can leave out line 12 with @ryantm's suggestion